processing/src/main/java/org/apache/druid/query/aggregation/SimpleDoubleAggregatorFactory.java

```diff
@@ -235,6 +244,19 @@ public String getExpression()
     return expression;
   }
 
+  @Override
+  public boolean canVectorize(ColumnInspector columnInspector)
+  {
+    if (fieldName != null) {
+      final ColumnCapabilities capabilities = columnInspector.getColumnCapabilities(fieldName);
+      return capabilities == null || ValueType.isNumeric(capabilities.getType());
+    }
+    if (expression != null) {
+      return fieldExpression.get().canVectorize(columnInspector);
+    }
+    return false;
+  }
+
```

Comment:
If I'm reading this correctly, this change isn't really needed in this PR, but it's some consolidation that was done across the various aggregator factories

Reply:
Ah yeah, I was just consolidating. Since these were the same across all 3 still, I've moved this into a new static `AggregatorUtil.canVectorize` method.